### ISSUE TITLE
bug: Globbing does not filter out directories

### ISSUE THREAD

**Issue by Timer** [FullHuman/purgecss#430]
PurgeCSS allows configuration like this:
```js
const purgecss = [
  "@fullhuman/postcss-purgecss",
  {
    content: ["./components/**/*", "./pages/**/*"],
    whitelist: ["html", "body", ":root"],
    whitelistPatterns: [/ag-(.*)/],
    defaultExtractor: (content) => content.match(/[\w-/:]+(?<!:)/g) || [],
  },
];
```

However, there's a subtle issue with the above code. The `content` blobs do not specify an extension, so **directories** will be matched.

This causes the build to fail with:
```
Error: EISDIR: illegal operation on a directory, read
```

However, changing the content blob to:

```js
content: ["./components/**/*.tsx", "./pages/**/*.tsx"]
```

... allows this to work.

---

Would it be worth adding the `nodir` option to the `glob` calls?

**Comment by Ffloriel** [comment]
Thanks for the suggestion, it's true it is easier to add `nodir` to the `glob` calls. I released this change in 2.3.0

**Comment by Timer** [comment]
Thank you so much 🚀
